# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/ipdl/ipdl.py
# Commit: 7b7819250f79
# Full Hash: 7b7819250f7965227783233af6de0e4e1b789af6
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2021-06-23 09:53:24
# Regressor Bug: 1706374
# File Overlap Count: 1
# Description:
#   Bug 1706374 - Part 11: Add NodeController component bridging IPC and Ports, r=handyman
#   
#   The NodeController and NodeChannel types act as the backbone connecting the
#   existing IPC logic and driving the ports routing code. Individual NodeChannel
#   objects wrap and respond to messages from IPC::Channel, and the NodeController
# ==============================================================================

diff -r 8bdac5eedf78 -r 7b7819250f79 ipc/ipdl/ipdl.py
--- a/ipc/ipdl/ipdl.py	Tue Jun 22 18:17:21 2021 +0000
+++ b/ipc/ipdl/ipdl.py	Tue Jun 22 18:17:22 2021 +0000
@@ -271,6 +271,22 @@
 
 print(
     """
+  case ACCEPT_INVITE_MESSAGE_TYPE:
+    return "ACCEPT_INVITE_MESSAGE";
+  case REQUEST_INTRODUCTION_MESSAGE_TYPE:
+    return "REQUEST_INTRODUCTION_MESSAGE";
+  case INTRODUCE_MESSAGE_TYPE:
+    return "INTRODUCE_MESSAGE";
+  case BROADCAST_MESSAGE_TYPE:
+    return "BROADCAST_MESSAGE";
+  case EVENT_MESSAGE_TYPE:
+    return "EVENT_MESSAGE";
+  case IMPENDING_SHUTDOWN_MESSAGE_TYPE:
+    return "IMPENDING_SHUTDOWN";
+  case BUILD_IDS_MATCH_MESSAGE_TYPE:
+    return "BUILD_IDS_MATCH_MESSAGE";
+  case BUILD_ID_MESSAGE_TYPE:
+    return "BUILD_ID_MESSAGE";
   case CHANNEL_OPENED_MESSAGE_TYPE:
     return "CHANNEL_OPENED_MESSAGE";
   case SHMEM_DESTROYED_MESSAGE_TYPE:
